src/foam/u2/theme/StandaloneTheme.js

```diff
@@ -0,0 +1,66 @@
+/**
+* @license
+* Copyright 2024 The FOAM Authors. All Rights Reserved.
+* http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+foam.CLASS({
+  package: 'foam.u2.theme',
+  name: 'StandaloneTheme',
+
+  documentation: `
+    A barebones theme object that can be used to theme non-nanos apps
+    Currently provides: 
+      - id for cssTokenOverrides
+      - name
+      - glyphs
+      - logos
+  `,
+
+
+  requires: [
+    'foam.u2.theme.ThemeGlyphs'
+  ],
+
+  properties: [
+    {
+      class: 'String',
+      name: 'id',
+      includeInDigest: true,
+      section: 'infoSection',
+      createVisibility: 'HIDDEN',
```

Comment:
Is visibility information needed?

Reply:
Not in this case, this theme object never goes to the DAO, this is a theme for when we dont use nanos